Resources/Locale/en-US/traits/traits.ftl

```diff
@@ -55,3 +55,8 @@ trait-name-Foreigner = Foreigner
 trait-description-Foreigner =
     For one reason or another you do not speak this station's primary language.
     Instead, you have a translator issued to you that only you can use.
+
+trait-name-Voracious = Voracious
+trait-description-Voracious =
+    Nothing gets between you and your food.
+    You eat food and drink drinks twice as fast.
```

Comment:
```suggestion
    Your endless consumption is twice as fast.
```

Reply:
"Your endless consumption of food and drinks is twice as fast."

Need to mention drinks because that's not immediately apparent by the name and the first line

Reply:
Resolved with [d5209a4](https://github.com/Simple-Station/Einstein-Engines/pull/668/commits/d5209a48c22f415fa0dd0110adf462b312fcc560)